Formula/s/sevenzip.rb

```diff
@@ -25,21 +25,24 @@ class Sevenzip < Formula
   end
 
   def install
+    mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
+    mk_suffix, directory = if OS.mac?
+      ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
+    else
+      ["gcc", "g"]
+    end
     cd "CPP/7zip/Bundles/Alone2" do
-      mac_suffix = Hardware::CPU.intel? ? "x64" : Hardware::CPU.arch
-      mk_suffix, directory = if OS.mac?
-        ["mac_#{mac_suffix}", "m_#{mac_suffix}"]
-      else
-        ["gcc", "g"]
-      end
-
       system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
 
       # Cherry pick the binary manually. This should be changed to something
       # like `make install' if the upstream adds an install target.
       # See: https://sourceforge.net/p/sevenzip/discussion/45797/thread/1d5b04f2f1/
       bin.install "b/#{directory}/7zz"
     end
+    cd "CPP/7zip/Bundles/Format7zF" do
+      system "make", "-f", "../../cmpl_#{mk_suffix}.mak", "DISABLE_RAR_COMPRESS=1"
+      lib.install "b/#{directory}/7z.so"
```

Comment:
Can you add a simple test for this to make sure it has been built and installed correctly?